### PR TITLE
SPM-1537: replace shell scripts

### PR DESCRIPTION
**Pull request by psegedy**
- replaced wait-for-services.sh
- db migration without shell scripts
- ~removed export of clowder variables as env vars~
- removed postgresql from image
- removed shell scripts depending on the functionality listed above

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Comment by psegedy** [comment]
doesn't work in ephemeral with clowder :/
```
time="2022-06-03T18:06:13Z" level=error msg=Panicked err="dial tcp 172.30.157.235:0: connect: no route to host" stack="goroutine 1 [running]:|runtime/debug.Stack()|\t/usr/lib/golang/src/runtime/debug/stack.go:24 +0x65|app/base/utils.LogPanics(0x1)|\t/go/src/app/base/utils/core.go:138 +0x58|panic({0x1050300, 0xc00047c7d0})|\t/usr/lib/golang/src/runtime/panic.go:1038 +0x215|app/database_admin.UpdateDB({0x7fffefe3ff85, 0x22})|\t/go/src/app/database_admin/update.go:84 +0x571|main.main()|\t/go/src/app/main.go:35 +0x10a|"
```

File: scripts/feed_db.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+import (
+	"app/base/database"
+	"io/ioutil"
+	"os"
+)
+
+func main() {
+	database.InitDB()
+	os.Setenv("WAIT_FOR_EMPTY_DB", "true")
```
[the remark this answers is not in the document]
Comment (psegedy) [reply scripts/feed_db.go:11]: hmm yeah I needed a different value than the value set in test.env, argument for DBWait would be better

**Comment by psegedy** [comment]
PR checks are failing with `E       fixture 'upload_system' not found`

**Review by MichaelMraka — APPROVED** [review]

**Pull request merged**